task5/src/main/java/com/clouway/core/SessionRepository.java

```diff
@@ -0,0 +1,21 @@
+package com.clouway.core;
+
+/**
+ * Created by Kristiyan Petkov  <kristiqn.l.petkov@gmail.com> on 02.06.16.
+ */
+public interface SessionRepository {
+
+  void create(Session session);
+
+  Session get(String sessionID);
+
+  void delete(String sessionID);
+
+  void deleteAll();
+
+  void cleanExpired();
+
+  int getOnlineUsersNumber();
```

Comment:
`getActiveSessions()` using `User` in the name is not correct for me. it is session repository not user repository.